src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -398,6 +403,40 @@ public void setJenkinsURL(String jenkinsURL) {
 	this.jenkinsURL = jenkinsURL;
 }
 
+  public String getRejectOfferDuration() {
+    if (rejectOfferDuration == null) {
+      return DEFAULT_REJECT_OFFER_DURATION;
+    } else {
+      return rejectOfferDuration;
+    }
+  }
+
+  public double getRejectOfferDurationDouble() {
+    return Double.parseDouble(getRejectOfferDuration());
+  }
+
+  public void setRejectOfferDuration(String rejectOfferDuration) {
+    try {
+      if (rejectOfferDuration == null) {
+        LOGGER.info("Missing rejectOfferDuration. Using default " + DEFAULT_REJECT_OFFER_DURATION + ".");
+        this.rejectOfferDuration = DEFAULT_REJECT_OFFER_DURATION;
+      } else {
+        double duration = Double.parseDouble(rejectOfferDuration);
+        if (duration >= 1000) {
```

Comment:
is it possible for the user to set a value < 1000 given the "min" condition in config.jelly?